bciers/apps/reporting/src/app/components/activities/generalStationaryCombustion.tsx

```diff
@@ -0,0 +1,186 @@
+"use client";
+import ActivityForm from "./ActivityForm";
+import FieldTemplate from "@bciers/components/form/fields/FieldTemplate";
+import NestedArrayFieldTemplate from "@bciers/components/form/fields/NestedArrayFieldTemplate";
+import SourceTypeBoxTemplate from "@bciers/components/form/fields/SourceTypeBoxTemplate";
+import CheckboxWidgetLeft from "@bciers/components/form/widgets/CheckboxWidgetLeft";
+import InlineFieldTemplate from "@bciers/components/form/fields/InlineFieldTemplate";
+
+interface Props {
+  activityData: {
+    activityId: number;
+    sourceTypeMap: { [key: number]: string };
+  };
+  reportDate: string;
+}
+
+// 🧩 Main component
+export default function GeneralStationaryCombustion({
+  activityData,
+  reportDate,
+}: Readonly<Props>) {
+  const uiSchema = {
```

Comment:
The UISchema can probably be declared statically outside the render function, to make the function short, and to avoid an (somewhat slightly more) expensive parse/assignment every render